### PR TITLE
Update navigation bar tint color and change eat now VC title

### PR DESCRIPTION
**Pull request by eappel**
Addresses #60.

### CODE REVIEW

File: Classes/Eat Now/Root View Controller/EatNowTableViewController.swift [hunk 1]
```diff
@@ -65,7 +65,7 @@ class EatNowTableViewController: UITableViewController, UISearchResultsUpdating,
         // TODO: make this a proxy and put it in another file
         navigationController?.view.backgroundColor = UIColor.whiteColor()
         navigationController?.navigationBar.translucent = false
-        navigationController?.navigationBar.barTintColor = UIColor.eateryBlue()
+        navigationController?.navigationBar.barTintColor = UIColor.blueColor()
```
Comment (alexzielenski) [inline Classes/Eat Now/Root View Controller/EatNowTableViewController.swift]: objectively bad code style

File: Other-Sources/AppDelegate.swift [hunk 2]
```diff
@@ -35,7 +35,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         // Set up view controllers
         let eatNow = EatNowTableViewController()
-        eatNow.title = "Eateries"
+        eatNow.title = "Places to Eat"
```
Comment (eappel) [inline Other-Sources/AppDelegate.swift]: To should be capitalized

Comment (eappel) [inline Other-Sources/AppDelegate.swift]: Fixed eb8ae60bb7e0332d3bc90c3516d0010702b078bb

**Comment by eappel** [comment]
:ship:

**Comment by eappel** [comment]
Merged 51b3df6c58ac90c829310944b9743fe37c29d14a

**Pull request closed without merging**